stylesheets/style.css

```
@@ -0,0 +1,30 @@
+body {
+  text-align: center;
+  font-family: Arial, sans-serif;
+  cursor: url("../images/pasta-icon.png"), auto;
+  background-color: #fffaf4;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  font-size: 2.5rem;
+}
+
+ul {
+  list-style-type: none;
+  list-style: none;
```

Instead of having redundant list styles, you can just use `list-style` set to `none` as well
```suggestion
  list-style: none;
```

---

I’ll remove the redundancy and keep only list-style: none; as it's enough to reset the list style. Thanks!